src/components/Modal/Modal.tsx

```diff
@@ -116,7 +118,9 @@ export function Modal(props: ModalProps) {
                     .df.fdc.wPx(width)
                     .mhPx(defaultMinHeight)
                     .if(isFixedHeight)
-                    .hPx(height).$
+                    .hPx(height)
+                    .if(sm)
+                    .vh100.add("width", "100vw").$
```

Comment:
**request** Can you additionally add in `maxh("none")` and `br0`. This way it really does take up the full screen. Right now we still hit the `maxh("90vh")` and see rounded corners, which to me look odd for this. I think we might as well remove those and take up the full screen. Thanks!